core/node/da_clients/src/eigen/sdk.rs

```diff
@@ -1,61 +1,127 @@
 use std::{str::FromStr, time::Duration};
 
 use secp256k1::{ecdsa::RecoverableSignature, SecretKey};
-use tokio::sync::mpsc;
+use tokio::{sync::mpsc, time::Instant};
 use tokio_stream::{wrappers::ReceiverStream, StreamExt};
 use tonic::{
     transport::{Channel, ClientTlsConfig, Endpoint},
     Streaming,
 };
-
+use zksync_config::configs::da_client::eigen::DisperserConfig;
+#[cfg(test)]
+use zksync_da_client::types::DAError;
+
+use super::{
+    blob_info::BlobInfo,
+    disperser::BlobInfo as DisperserBlobInfo,
+    verifier::{Verifier, VerifierConfig},
+};
 use crate::eigen::{
-    disperser,
+    blob_info,
     disperser::{
+        self,
         authenticated_request::Payload::{AuthenticationData, DisperseRequest},
         disperser_client::DisperserClient,
-        AuthenticatedReply, BlobAuthHeader, BlobVerificationProof, DisperseBlobReply,
+        AuthenticatedReply, BlobAuthHeader, DisperseBlobReply,
     },
 };
 
 #[derive(Debug, Clone)]
-pub struct RawEigenClient {
+pub(crate) struct RawEigenClient {
     client: DisperserClient<Channel>,
-    polling_interval: Duration,
     private_key: SecretKey,
-    account_id: String,
+    pub config: DisperserConfig,
+    verifier: Verifier,
 }
 
 pub(crate) const DATA_CHUNK_SIZE: usize = 32;
 
 impl RawEigenClient {
     pub(crate) const BUFFER_SIZE: usize = 1000;
 
-    pub async fn new(
-        rpc_node_url: String,
-        inclusion_polling_interval_ms: u64,
-        private_key: SecretKey,
-    ) -> anyhow::Result<Self> {
+    pub async fn new(private_key: SecretKey, config: DisperserConfig) -> anyhow::Result<Self> {
         let endpoint =
-            Endpoint::from_str(rpc_node_url.as_str())?.tls_config(ClientTlsConfig::new())?;
+            Endpoint::from_str(config.disperser_rpc.as_str())?.tls_config(ClientTlsConfig::new())?;
         let client = DisperserClient::connect(endpoint)
             .await
             .map_err(|e| anyhow::anyhow!("Failed to connect to Disperser server: {}", e))?;
-        let polling_interval = Duration::from_millis(inclusion_polling_interval_ms);
-
-        let account_id = get_account_id(&private_key);
 
+        let verifier_config = VerifierConfig {
+            verify_certs: true,
+            rpc_url: config.eigenda_eth_rpc.clone(),
+            svc_manager_addr: config.eigenda_svc_manager_address.clone(),
+            max_blob_size: config.blob_size_limit,
+            path_to_points: config.path_to_points.clone(),
```

Comment:
Do I understand correctly that these points have to be present every time this code is run? This would add an ops load that we would ideally want to avoid. Is there a way to get rid of the points file?